RELEASE_NOTES.md

```diff
@@ -85,6 +85,14 @@ making a backup is strongly advised.
   with different default settings, because input color profiles and 
   demosaicing need at least a rough white balancing early in the pipe.
 
+  Map view has evolved. Images close to each other are now grouped and a count of
+  grouped images is displayed on the bottom-left corner. This provides better performances
+  when the map covers thousands of images. Scrolling over a thumb scrolls through
+  the images of the group. Dragging of images has also been improved.
```

Comment:
Maybe talk here about the shift drag for a group?

Reply:
ok